src/celeritas/global/KernelContextException.cc

```diff
@@ -133,6 +135,16 @@ void KernelContextException::initialize(CoreTrackView const& core)
             }
             surface_ = geo.surface_id();
         }
+
+        if (!celeritas::getenv("CELERITAS_EXCEPTION_DEBUG").empty())
```

Comment:
When is `void KernelContextException::initialize(CoreTrackView const& core)` called?

Reply:
During the constructor of the KernelContextException...